gpflow/utilities/utilities.py

```diff
@@ -420,7 +420,10 @@ def traverse_module(
             new_acc = (f"{path}['{term_idx}']", new_state)
             new_state = traverse_module(subterm, new_acc, update_cb, target_types)
     elif isinstance(m, tf.Module):
-        for name, submodule in vars(m).items():
+        module_vars = vars(m)
+        if isinstance(m, tfp.bijectors.Bijector):
+            module_vars = {name: module_vars[name] for name in module_vars if name != "_parameters"}
```

Comment:
nit:
I feel it's more idiomatic to use the items iterator, so you dont need to check the dict with the value you are looking up is the same one as your are iterating through

 ```suggestion
             module_vars = {name: var for name, var in module_vars.items() if name != "_parameters"}
```